AsyncDisplayKit/Details/_ASDisplayView.mm

```diff
@@ -202,6 +202,15 @@ - (void)willRemoveSubview:(UIView *)subview
 #endif
 }
 
+- (CGSize)sizeThatFits:(CGSize)size
```

Comment:
Unfortunately the way it's currently implemented will have side effects as you know if we call `layoutThatFits:` that will create an internal pending layout :(